dotcom-rendering/src/paletteDeclarations.ts

```diff
@@ -4892,6 +4892,39 @@ const staffPickBadgeTextLight: PaletteFunction = () =>
 	sourcePalette.neutral[100];
 const staffPickBadgeTextDark: PaletteFunction = () => sourcePalette.neutral[7];
 
+const speechBubbleBackgroundLight: PaletteFunction = ({ theme, design }) => {
+	if (theme === Pillar.News && design === ArticleDesign.Analysis) {
+		return sourcePalette.news[300];
+	}
+
+	switch (theme) {
+		case Pillar.News: {
+			return sourcePalette.news[400];
+		}
+		case Pillar.Opinion: {
+			return sourcePalette.opinion[400];
```

Comment:
This used to be `sourcePalette.opinion[300]`, however this has been deprecated in favour of `sourcePalette.opinion[400]`. Both colours are the same: #C74600.